### DISCUSSION TITLE
级联地址无法显示中文，且无法下拉选择

### DISCUSSION THREAD

**Discussion by tangtang09** [alibaba/formily#1692] [Q&A 2.x]
https://codesandbox.io/s/aged-https-3kgum?file=/App.tsx
比如：北京市/市辖区/朝阳区，设置值显示的是数字地标110000 / 110100 / 110105，且无法下拉选择

**Comment by janryWang** [comment]
没懂你要说明啥问题，具体复现步骤是啥？
